modules/rank-eval/src/main/java/org/elasticsearch/index/rankeval/ReciprocalRank.java

```diff
@@ -45,7 +45,7 @@
 public class ReciprocalRank implements RankedListQualityMetric {
 
     public static final String NAME = "reciprocal_rank";
-    public static final int DEFAULT_MAX_ACCEPTABLE_RANK = 10;
```

Comment:
Why not control this through the search request size parameter?

Also reading through this code I was wondering where we had gotten the information on how to treat the edge case of "result set doesn't contain any relevant search results". Would it make sense to track this information in some comment?